WordPress/Classes/ViewRelated/People/PersonViewController.swift

```diff
@@ -190,9 +190,12 @@ final class PersonViewController: UITableViewController {
             lastNameIndexPath,
             displayNameIndexPath
         ])
-        model.append([
-            removeIndexPath
-        ])
+
+        if isRemoveEnabled {
```

Comment:
Without this conditional the "Remove [username]" section still exists although the cell is hidden, so it leaves a large vertical gap above the badge.

<img width="873" alt="Screen Shot 2023-01-03 at 17 02 56" src="https://user-images.githubusercontent.com/2092798/210452757-e060de40-3b55-480e-b806-31ad70592d47.png">